packages/block-library/src/media-text/edit.native.js

```diff
@@ -99,13 +98,8 @@ class MediaTextEdit extends Component {
 		if ( mediaType === 'image' && media.sizes ) {
 			// Try the "large" size URL, falling back to the "full" size URL below.
 			src =
-				get( media, [ 'sizes', 'large', 'url' ] ) ||
-				get( media, [
-					'media_details',
-					'sizes',
-					'large',
-					'source_url',
-				] );
+				media?.sizes?.large?.url ||
```

Comment:
Seems like `media` should be defined. Shall we remove the first `?`?

Reply:
Good call 👍 

Removed in https://github.com/WordPress/gutenberg/pull/48491/commits/dc1ab15bfdcfa89e65ac19ed48a7c46dbab9ba86. ✅ 

I also removed the second one, since this is wrapped in an `if()` that ensures that `media.sizes` will also be defined.